package exceptions

class UserAlreadyExistsException: Exception()